lib/fastlane/plugin/firebase_app_distribution/helper/upload_status_response.rb

```diff
@@ -3,35 +3,50 @@ def initialize(response_json_hash)
     @response_json_hash = response_json_hash
   end
 
-  def status
-    @response_json_hash[:status]
+  def done
+    !!@response_json_hash[:done]
   end
 
-  def success?
-    status == 'SUCCESS'
+  def response
+    @response_json_hash[:response]
   end
 
-  def in_progress?
-    status == 'IN_PROGRESS'
+  def release
+    response ? response[:release] : nil
   end
 
-  def error?
-    status == 'ERROR'
+  def release_id
+    if release
+      split_name = release[:name].split('/')
+      split_name[split_name.length - 1]
+    end
   end
 
-  def already_uploaded?
-    status == 'ALREADY_UPLOADED'
+  def status
+    response ? response[:result] : nil
   end
 
-  def release_hash
-    @response_json_hash[:release]
+  def error
+    @response_json_hash[:error]
   end
 
-  def release_id
-    release_hash ? release_hash[:id] : nil
+  def error_message
+    error ? error[:message] : nil
+  end
+
+  def success?
+    done && status == 'RELEASE_CREATED'
```

Comment:
Wouldn't it also be successful if `status = 'RELEASE_UPDATED'`?

Reply:
Good catch, I updated this to just check for a non-nil `release` instead.